### DISCUSSION TITLE
No downloads possible

### DISCUSSION THREAD

**Discussion by Deitenbeck** [aclap-dev/video-downloadhelper#1571] [Help and Q&A]
### Your question

I've just purchased the licence for the full version of download helper. But it cannot download YT videos anymore. I always get an error message.

### Addon & CoApp version and details

![grafik](https://github.com/user-attachments/assets/1a788da9-0636-4dca-8e4a-7e30d4b25652)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

On a hunch . . . accept this as the guess it is . . . you might want to look at your antivirus.  Click the link to the Wiki.  This link appears at the top of every page of this forum.  There is a discussion in there on the subject of antivirus.  Read that & see if that's relevant to you.
